content/pages/schedule.rst

```diff
@@ -2,26 +2,23 @@ Schedule
 ########
 :slug: schedule
 
------------------
+(NOTE: each session is either 25 or 50 minutes long – please don’t run over)
 
-================= ==================
-Time              Event
-9:00am - 9:30am   Morning break
-9:30am - 10:20am  Session Planning
-10:20am - 10:30am Session Scheduling
-10:30am - 11:20am Session 1
-11:30am - 12:20pm Session 2
-12:30pm - 1:30pm  Lunch
-1:30pm - 2:20pm   Session 3
-2:30pm - 3:20pm   Session 4
-3:30pm - 4:00pm   Snack Break
-4:00pm - 4:50pm   Session 5
-5:00pm - 5:50pm   Session 6
-================= ==================
+=================   ==================
+Time                Event
+9:00 – 10:00        Setup
```

Comment:
I was using the one from the google doc: https://docs.google.com/a/osuosl.org/document/d/1CWtkoamNHjX_515ux1EvVT__i7_kdW_DHxeZap41vDQ/edit

Out of curiosity, why do we have the google doc if so much of the information on it is wrong?